debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/xstream/OracleVersion.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.oracle.xstream;
+
+import io.debezium.config.EnumeratedValue;
+import oracle.streams.XStreamUtility;
+
+/**
+ * This enum class includes Oracle 11 and 12 major versions
+ * It returns position version based on the Oracle release.
+ * Position version get used in conversion SCN (system change number) into position
+ */
+public enum OracleVersion implements EnumeratedValue {
+
+    V11("11"),
+    V12Plus("12+");
+    private final String version;
+
+    OracleVersion(String version) {
+        this.version = version;
+    }
+
+    @Override
+    public String getValue() {
+        return version;
+    }
+
+    public int getPosVersion() {
+        switch(version) {
+            case "11":
```

Comment:
Do you know whether it'd work with 10, too?

Reply:
I don't know , but my general feeling that nobody should use 10 since 11, 12, 18 and 19 were introduced.

Reply:
Ok, thanks!